integration-cli/docker_cli_links_test.go

```diff
@@ -2,18 +2,16 @@ package main
 
 import (
 	"fmt"
-	"os/exec"
+	"github.com/go-check/check"
 	"reflect"
 	"regexp"
 	"strings"
 	"time"
-
-	"github.com/go-check/check"
 )
 
 func (s *DockerSuite) TestLinksPingUnlinkedContainers(c *check.C) {
-	runCmd := exec.Command(dockerBinary, "run", "--rm", "busybox", "sh", "-c", "ping -c 1 alias1 -W 1 && ping -c 1 alias2 -W 1")
-	exitCode, err := runCommand(runCmd)
+
+	_, exitCode, err := dockerCmdWithError(c, "run", "--rm", "busybox", "sh", "-c", "ping -c 1 alias1 -W 1 && ping -c 1 alias2 -W 1")
```

Comment:
I think in this test we can just use `dockerCmd` since the it's checking error with the exitCode
```
if exitCode == 0 {
 		c.Fatal("run ping did not fail")
 	} else if exitCode != 1 {
 		c.Fatalf("run ping failed with errors: %v", err)
 	}
```
so just `dockerCmd` would work, WDYT?